schema-v1alpha1.yaml

```diff
@@ -1,8 +1,98 @@
 openAPIV3Schema:
-  type: object
+  description: SealedSecret is the K8s representation of a "sealed Secret" -
+    a regular k8s Secret that has been sealed (encrypted) using the controller's
```

Comment:
How can we omit `object` here? and why?

Reply:
It was a typo at copy&pasting. Should be there now.